lib/tic_tac_toe.ex

```diff
@@ -32,9 +32,9 @@ defmodule TicTacToe do
          %{players: [first_player, second_player], board: board} = state
        ) do
     first_player = Player.selection(first_player, board)
-    {selection, token} = Player.get(first_player)
+    player_properties = Player.get(first_player)
```

Comment:
I am wondering about this naming. Is this return "player properties" or is it returning some sort of "move"?

Reply:
likely a move I'll adjust